lib/next_ls/runtime.ex

```diff
@@ -136,8 +136,8 @@ defmodule NextLS.Runtime do
           |> Path.join("monkey/_next_ls_private_compiler.ex")
           |> then(&:rpc.call(node, Code, :compile_file, [&1]))
           |> tap(fn
-            {:badrpc, :EXIT, {error, _}} ->
```

Comment:
Looking at the `:rpc` docs, I don't think this would ever match, since it should be a 2-tuple. [Docs](https://www.erlang.org/doc/man/rpc#call-4)